requirements/infrastructure-requirements.md

```diff
@@ -197,15 +199,18 @@ Users must never get a 'certificate expired' error in their browser,
 so the infrastructure must check expirations and renew certificates automatically,
 for instance using [Let's Encrypt](https://letsencrypt.org).
 
-### 13. The infrastructure *MUST* back up all application data from delivered Clariah services at an agreed upon interval and has working restore functionality. ([REL](#rel))
+### 13. The infrastructure *MUST* back up all application data from deployed services at an agreed upon interval. ([REL](#rel))
 
 Data loss is unacceptable, especially when that data has been provided by users in
 [stateful applications](#3-the-infrastructure-must-be-able-to-run-_stateful-applications_).
-Both backup and restore functionality must be tested.
+Both backup and restore functionality *MUST* be tested.
 
 ### 14. The infrastructure *MUST* be _GDPR-compliant_, for instance in the way it stores data.
 
 If the infrastructure stores personal data, for instance logs, this must be done in a GDPR-compliant manner.
+This entails, amongst others, that the infrastructure *SHOULD* give users the
+option to download all their user data and *MUST* give users the option to
+remove their data.
```

Comment:
While this point was already in the Infrastructure Requirements, I’m no longer sure if this is an infrastructural concern. Isn’t it up to the software applications to offer inspection/removal functionality? Of course, we still want to prevent that the infrastructure needlessly logs any PII and, if it does need to capture it, offers removal functionality to the applications that run on the infrastructure.

```suggestion
The infrastructure *MUST* store personally identifiable information (PII) only if necessary. 
If the infrastructure stores PII, it *MUST* give users (or software applications on behalf of users) the option to remove their data
and it *SHOULD* give users the option to download their data.
```

Reply:
I agree, that sounds clearer indeed!